src/plugins/cache/CMakeLists.txt

```diff
@@ -14,6 +15,13 @@ if (NOT_INCLUDED)
 	return ()
 endif (NOT_INCLUDED)
 
+safe_check_symbol_exists (nftw "ftw.h" HAVE_NFTW)
+
+if (NOT HAVE_NFTW)
+	message ("nftw (ftw.h) not found, excluding the cache plugin")
+	return ()
```

Comment:
Please exclude the plugin properly (with remove_plugin), otherwise it is not listed in the excluded plugins.

Reply:
After adding the check with `safe_check_symbol_exists (nftw "ftw.h" HAVE_NFTW)`, it does not find nftw any more (on debian 9, where it works fine without the check).